packages/react-component-library/src/components/Timeline/TimelineEvent.tsx

```diff
@@ -118,15 +186,27 @@ export const TimelineEvent: React.FC<TimelineEventProps> = ({
   const {
     offset: offsetPx,
```

Comment:
These renamed properties could be below the ones that do not need renaming on lines 190-193. This would be consistent with other areas of the code.

I always accepted this as a coding standard. Maybe we need a lint rule if we agree on this?